internal/httperror/httperror.go

```diff
@@ -602,6 +659,8 @@ func DecodeSigningKeyError(err string) HTTPError {
 	switch getCallerFunctionName() {
 	case "SetIntermediateCertificate":
 		errorCode = "CPKISI009"
+	case "CreateSSHCertificate":
+		errorCode = "CSSHCC012"
```

Comment:
Should this error code be the same as the one below?

Reply:
No, I'll figure out the appropriate numbers and fix them... That was an oversight